src/InvokePlaster.ps1

```diff
@@ -70,7 +70,7 @@ function Invoke-Plaster {
             # grabbing the parameter's value which is static.
             $templateAbsolutePath = $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath($TemplatePath)
             if (!(Test-Path -LiteralPath $templateAbsolutePath -PathType Container)) {
-                throw ($LocalizedData.ErrorTemplatePathIsInvalid_F1 -f $templateAbsolutePath)
+                Write-Error -ErrorAction $ErrorActionPreference -Message ($LocalizedData.ErrorTemplatePathIsInvalid_F1 -f $templateAbsolutePath)
```

Comment:
Hi,
I based my scripts on the presence of these exceptions :-)
I do not think this change is necessary.

Reply:
The commands will still throw an exception, you should not be impacted at all

The change from throw to `Write-Error -EA Stop` is to further encapsulate error information.

If you use throw internal information is leaked from the function while Write-Error ensures it does not.

For comparison throw:
```
function Error { throw 'bad thing!' }
Error
bad thing!
At line:1 char:18
+ function Error { throw 'bad thing!' }
+                  ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (bad thing!:String) [], RuntimeException
    + FullyQualifiedErrorId : bad thing!
```

And now Write-Error:
```
function Error { Write-Error -Message 'bad thing!' -ErrorAction Stop }
Error
Error : bad thing!
At line:1 char:1
+ Error
+ ~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Error
```

Notice how throw leaks internal details while Write-Error does not.